cli/core/findStalePods.go

```diff
@@ -204,30 +209,50 @@ func FindStaleEigenpods(ctx context.Context, eth *ethclient.Client, nodeUrl stri
 		}
 	}
 
-	if verbose {
-		log.Printf("%d EigenPods were slashed\n", len(slashedEigenpods))
-	}
-
 	allValidatorBalances, err := beaconState.ValidatorBalances()
 	if err != nil {
 		return nil, err
 	}
 
-	var unhealthyEigenpods map[string]bool = make(map[string]bool)
-	for _, validator := range allActiveSlashedValidatorsBelongingToEigenpods {
-		balance := allValidatorBalances[validator.Index]
-		pod := validatorToPod[validator.Index]
-		executionBalance := cache.PodOwnerShares[pod].Shares
-		if executionBalance == 0 {
-			continue
+	totalAssetsGweiByEigenpod := utils.Reduce(keys(slashedEigenpods), func(allBalances map[string]phase0.Gwei, eigenpod string) map[string]phase0.Gwei {
+		// total assets of an eigenpod are determined as;
+		//	SUM(
+		//		- native ETH in the pod
+		//		- any active validators and their associated balances
+		// 	)
+		allEigenpodsForValidator := utils.Filter(allValidatorsWithIndices, func(v ValidatorWithIndex) bool {
+			withdrawal := executionWithdrawalAddress(v.Validator.WithdrawalCredentials)
+			return withdrawal != nil && strings.EqualFold(*withdrawal, eigenpod)
+		})
+
+		allValidatorBalancesSummed := utils.Reduce(allEigenpodsForValidator, func(accum phase0.Gwei, validator ValidatorWithIndex) phase0.Gwei {
+			return accum + allValidatorBalances[validator.Index]
+		}, phase0.Gwei(0))
+
+		balance := phase0.Gwei(cache.PodOwnerShares[eigenpod].NativeETH) + allValidatorBalancesSummed
+		allBalances[eigenpod] = phase0.Gwei(balance)
+		return allBalances
+	}, map[string]phase0.Gwei{})
```

Comment:
can i sum all the validator balances, plus the native eth?
or do I also need to filter the validators by which ones are active?